script/Notebooks.toml

```diff
@@ -0,0 +1,8 @@
+[MNIST]
+path = "vision/mnist"
+notebook = "mlp.jl"
+deps = ["vae.jl"]
+
+[CHAR-RNN]
```

Comment:
maybe we should just lower case these so they're easier to type (mnist was only all caps because it's an acronym anyway).